dashboards/node.libsonnet

```diff
@@ -135,10 +135,10 @@ local gauge = promgrafonnet.gauge;
           format='percentunit',
         )
         .addTarget(prometheus.target(
-          'node:node_filesystem_usage:{%(clusterLabel)s="$cluster", instance="$instance"}' % $._config, legendFormat='disk used'
+          'node:node_filesystem_usage:{%(clusterLabel)s="$cluster", instance="$instance"}' % $._config, legendFormat='{{device}} disk used'
         ))
         .addTarget(prometheus.target(
-          'node:node_filesystem_usage:{%(clusterLabel)s="$cluster", instance="$instance"}' % $._config, legendFormat='disk free'
+          'node:node_filesystem_usage:{%(clusterLabel)s="$cluster", instance="$instance"}' % $._config, legendFormat='{{device}} disk free'
```

Comment:
I seem to remember that node exporter recently changed this label, can we make sure it’s the label exposed by latest node exporter?

Reply:
I'm running node_exporter 0.18.1 on my VM and I have the device label available. It's also the label you're using in the recording rule, so I think it should be correct.

Reply:
They changed the label for network metrics from `interface` to `device` in [0.18.0](https://github.com/prometheus/node_exporter/releases/tag/v0.18.0). That might be what you remember.

Reply:
Yes, thanks for confirming!